src/components/CollectionComponents.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Stack,
+  Tooltip,
+} from "@mui/material";
+import React from "react";
+
+type EavaRequest = {
+  laserDwell: number;
+  expTime: number;
+};
+
+/**
+ * Calculate laser delay for EAVA (Excite And Visit Again) pump probe settings.
+ * These are the options labeles as "repeat#" in the UI, the number after repeat
+ * indicating how many rows are pumped at a time.
+ *
+ * @param {number} laserDwell - laser exposure time
+ * @param {number} expTime - collection exposure time for each window
+ * @param {number} factor - number of rows pumped at the time, multiplied by 2
+ * @returns {number} The laser delay to set
+ */
+function calculateEAVA(
+  laserDwell: number,
+  expTime: number,
+  factor: number
+): number {
+  const movetime: number = 0.008;
```

Comment:
```suggestion
  const moveTime: number = 0.008;
```
Also, where does this come from - is it possible to calculate from PV's instead or will it always have this value

Reply:
Could we have a comment to say that? It may help to fix issues later on if the setup ever changes